Mathlib/Algebra/Divisibility/Ring.lean

```diff
@@ -0,0 +1,81 @@
+/-
+Copyright (c) 2023 Oliver Nash. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Oliver Nash
+-/
+import Mathlib.Algebra.Divisibility.Basic
+import Mathlib.Data.Nat.Choose.Sum
+import Mathlib.Algebra.BigOperators.Ring
```

Comment:
I'm puzzled by this import, there don't seem to be any big operators in this file.

Reply:
This import was included because the lemma `Finset.dvd_sum` (which I need) resides in `Mathlib.Algebra.BigOperators.Ring` (though I actually think the new file which I'm proposing might be a better home).

However I've dropped it from the list because I pick it up transitively anyway.